api/src/main/java/jakarta/enterprise/inject/spi/CDI.java

```diff
@@ -152,8 +152,15 @@ private static void findAllProviders() {
     /**
      * Get the CDI BeanManager for the current context
      *
-     * @return the BeanManager
+     * @return the {@link BeanManager}
      */
     public abstract BeanManager getBeanManager();
 
+    /**
+     * Get the CDI {@link BeanManagerLite} for the current context.
+     *
+     * @return the {@link BeanManagerLite}
+     */
+    public abstract BeanManagerLite getBeanManagerLite();
```

Comment:
I'm thinking if maybe we shouldn't do

```java
public BeanManagerLite getBeanManagerLite() {
    return getBeanManager();
}
```

for backwards compatibility?

Implementations that don't implement CDI Full would have to override this method explicitly. CDI Full implementations don't have to care.

Reply:
Yeah, I came to the same conclusion yesterday evening! :)
Will update the PR in a bit.